src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts

```diff
@@ -340,6 +340,8 @@ export class VolumesListTableConfig implements InputTableConf {
     if (rowData.type === 'zpool') {
 
       actions.push({
+        id: rowData.name,
+        name: 'export_disconnect',
```

Comment:
This should match the label precisely, so maybe:
```suggestion
        name: 'Export/Disconnect',
        label: T(rowData.name),
```
For that matter, menu options ought to all be in the global helptext files, but maybe we haven't done that yet.

Reply:
Agree the actual button text ought to be in the helptext files, but those will be translated. I'm guessing we'd like the Ids to be static for automation purposes?